main.py

```diff
@@ -284,8 +287,33 @@ def search_key():
             windowSurface.blit(KEYPIC,box)
             pygame.display.flip()
             pygame.time.delay(1000)
+	    show_end_screen()
 
         pygame.display.flip()
 
+def show_end_screen():
+    BG=pygame.image.load('Images/jail background.jpg')
+    windowSurface.blit(pygame.transform.scale(BG,(size)),(0,0))
+    backtext = pygame.font.SysFont("liberationserif",40)
+    textSurf, textRect = text_objects("Dare to play again!!", backtext)

```

Comment:
I think we can do a better text than 'Dare to play again!!', please change this or scrap